recipes/chembl_webresource_client/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "chembl_webresource_client" %}
+{% set version = "0.9.31" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/5d/43/bf0a4d475be9b16f1cbecf0dfe5cb2652305ad239d726a9238d1c4d1713d/chembl-webresource-client-0.9.31.tar.gz
+  sha256: 7a51a158271f569c1b20354541ee10988f060bcdbe1ce9e76f0a0d76e57beb10
+
+build:
+  number: 0
+  skip: True  # [win]
```

I think this recipe can be ``noarch``, I think if you add the ``entry_points`` that should be fine
```suggestion
  noarch: python
  entry_points:    
    - chembl_ids=chembl_webresource_client.scripts.chembl_ids:main
    - chembl_sim=chembl_webresource_client.scripts.chembl_sim:main
    - chembl_sub=chembl_webresource_client.scripts.chembl_sub:main
    - chembl_m2t=chembl_webresource_client.scripts.chembl_m2t:main
    - chembl_t2m=chembl_webresource_client.scripts.chembl_t2m:main
    - chembl_act=chembl_webresource_client.scripts.chembl_act:main
```

---

Sure, let's try